pkg/util/bucket_util.go

```diff
@@ -24,7 +24,7 @@ import (
 
 func GetBucketAndItem(context string) (string, string) {
 	split := strings.SplitN(context, "/", 2)
-	if len(split) == 2 {
+	if len(split) == 2 && split[1] != "" {
```

Comment:
For example, if `context` is `test/`, itemName will be `""` and I think that is not how it is supposed to work, so I fixed it!

Reply:
Thanks!